core/scss/components/brand-bar/_brand-bar.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+//
+// Brand Bar
+//
+// Stanford brand bar with the wordmark logo that links to the Stanford homepage. The brand bar sits at the very top of a website.
+//
+// .su-brand-bar - Default brand bar with white wordmark on cardinal red background
```

Comment:
You do not need to call out the default variant as the templates will handle rendering it.

Reply:
Remove this line.

Reply:
Done.